vendor/bat-native-ads/src/bat/ads/internal/account/account.cc

```diff
@@ -64,27 +64,24 @@ void Account::OnPrefChanged(const std::string& path) {
   }
 }
 
-bool Account::SetWallet(const std::string& id, const std::string& seed) {
-  const WalletInfo& last_wallet = wallet_->Get();
+void Account::SetWallet(const std::string& id, const std::string& seed) {
+  const WalletInfo& last_wallet = GetWallet();
```

Comment:
as I understand this should be a copy, so we don't need reference here
maybe should rename the variable to `last_wallet_copy`

Reply:
Fixed